src/components/FileUploader/FileUploader-story.js

```diff
@@ -6,6 +6,38 @@ import FileUploader, { FileUploaderButton } from '../FileUploader';
 import FileUploaderSkeleton from '../FileUploader/FileUploader.Skeleton';
 import Button from '../Button';
 
+class App extends React.Component {
+  upload = ({ file }) =>
+    fetch('https://jsonplaceholder.typicode.com/posts/', {
```

Comment:
Rather than creating a service dependency here, we may want to consider something like [`fetch-mock`](https://www.npmjs.com/package/fetch-mock).